packages/embark-ens/src/index.js

```diff
@@ -312,15 +340,16 @@ class ENS {
       }
 
       const reverseNode = namehash.hash(address.toLowerCase().substr(2) + reverseAddrSuffix);
-      this.registerSubDomain(defaultAccount, subDomainName, reverseNode, address, secureSend, callback);
+      this.registerSubDomain(defaultAccount, subDomainName, reverseNode, address, secureSend, (err, result) => {
```

Comment:
`this.registerSubDomain(defaultAccount, subDomainName, reverseNode, address, secureSend, callback)`